extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -40,12 +51,18 @@ core::Property ListenHTTP::SSLCertificate("SSL Certificate", "File containing PE
 core::Property ListenHTTP::SSLCertificateAuthority("SSL Certificate Authority", "File containing trusted PEM-formatted certificates", "");
 
 core::Property ListenHTTP::SSLVerifyPeer(
-    core::PropertyBuilder::createProperty("SSL Verify Peer")->withDescription("Whether or not to verify the client's certificate (yes/no)")->isRequired(false)->withAllowableValue<std::string>("yes")
-        ->withAllowableValue("no")->withDefaultValue("no")->build());
+    core::PropertyBuilder::createProperty("SSL Verify Peer")->
+    withDescription("Whether or not to verify the client's certificate (yes/no)")->
```

Comment:
I'm not against this style change at all and the removal of the linted code, but let's move that to a separate PR so it can be tracked better through git.

Reply:
* sorry in the future. leave these as-is

Reply:
Not "just" style, changed these to use allowableValue**s** to avoid calling _withAllowableValue_ many times for the same prop.

Reply:
That is a fluent API.